source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -251,24 +273,31 @@ const SummaryList: React.FC<Props> = ({
         ...
   ]
   */
-  const map: Map<string, List> = new Map();
+
+  const itemIdmap: Map<string, List> = new Map();
   itemsWithAnswers.forEach((item: SummaryListItem) => {
-    if (!map.has(item.id)) {
-      map.set(item.id, {
+    // Group answers by item
+    if (!itemIdmap.has(item.id)) {
+      itemIdmap.set(item.id, {
         items: [item],
         answers: answers[item.id] ?? "",
       });
     } else {
-      map.get(item.id)?.items.push(item);
+      itemIdmap.get(item.id)?.items.push(item);
     }
   });
 
+  useEffect(() => {
+    sortAnswers(itemsWithAnswers);
```

Comment:
Why is this wrapped in an `useEffect`? `itemsWithAnswers` is not stateful data (although it probably should be).